### DISCUSSION TITLE
Welcome to eInvoicing-EN16931 Discussions!

### DISCUSSION THREAD

**Discussion by midran** [ConnectingEurope/eInvoicing-EN16931#391] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by leonardehrenfried** [comment]
Thanks for this!

Let me introduce myself: I'm a freelance software developer and I'm primarily interested in streamlining my accounting. However, I have lots of experience with another CEN XML standard, NeTEx, which has not been very successful at actually standardising anything. I would be interested to hear what the experiences of using EN16931 internationally are.

Is there a guideline to which questions are and aren't on topic for this forum?

**Reply by phax** [reply]
@leonardehrenfried  I would even recommend you create a new topic for that, at https://github.com/ConnectingEurope/eInvoicing-EN16931/discussions and not in this thread - for increased visibility...
